tools/src/main/java/org/apache/kafka/tools/TransactionsCommand.java

```diff
@@ -436,16 +436,25 @@ public String name() {
 
         @Override
         public void addSubparser(Subparsers subparsers) {
-            subparsers.addParser(name())
+            Subparser subparser = subparsers.addParser(name())
                 .help("list transactions");
+
+            subparser.addArgument("--duration-filter")
+                    .help("filter duration of transaction in ms, only transactions running longer than this duration will be returned")
```

Comment:
we should mention the default/value that gives all running transactions. Providing 0 and getting a ton of transactions might be confusing.

Reply:
Updated the help message in [81173c8](https://github.com/apache/kafka/pull/15384/commits/81173c89dbf50a98671694f99a051dad1e0b1821)